### DISCUSSION TITLE
Forget remembered variables

### DISCUSSION THREAD

**Discussion by tntsoft** [inertiajs/inertia#971] [Help]
Hi,

This seems to be a no brainer, but I can't find anything on the web about it. 

I have a series of variables remembered this way:
```
 remember  : {
        data: [
            'var1',
            'var2',
            'var3',
        ],
        key: 'Test'
    },

```

I'd simply like to forget everything with the key 'Test' associated. I'd want to Forget the variables because it's a Wizard form, and when launching th Wizard form again (with other parameters) I need to remove all the remembered variables. I can do it either on the frontend (vue2) or in the backend (Laravel/PHP).

Any help would be greatly appreciated!

**Comment by ajnsn** [comment]
Hey @tntsoft 
afaik there's no direct "forget" method. But Inertia uses the [History API](https://developer.mozilla.org/en-US/docs/Web/API/History). You should be able to manipulate it directly by calling [replaceState](https://developer.mozilla.org/en-US/docs/Web/API/History/replaceState) and overwrite/null the remembered sate.

Another option could be to use `Inertia.remember`, see [documentation over here](https://inertiajs.com/remembering-state#manually-saving-state).

Haven't tried this though, just a theory. Hope this helps!
